content/SCALE/SCALETutorials/Credentials/AdminRoles.md

```diff
@@ -80,25 +80,35 @@ When disabling a password for UI login, it is also disabled for SSH access.
 
 ## Accessing the System Through an SSH Session
 
-To enable SSH to access the system as an admin user (or for root):
+To enable SSH access to the system as an admin user (or root user), you must first configure the SSH service.
 
-1. Configure the SSH service.
+1. Go to **System > Services**, then click <span class="iconify" data-icon="mdi:pencil"></span> (**Edit**) for the SSH service.
 
-   a. Go to **System > Services**, then select **Configure** for the SSH service.
+2. In the **Password Login Groups** field, enter the groups you want to enable for password authentication (**truenas_admin**, **root**, etc.).
```

Comment:
not parallel construction and a step procedure should start with the verb action
```suggestion
2.  Enter the groups (**truenas_admin**, **root**, etc.)  you want to enable for password authentication in the **Password Login Groups** field.
```